openebs/pkg/provisioner/cas_provision.go

```diff
@@ -139,11 +139,6 @@ func (p *openEBSCASProvisioner) Provision(options controller.VolumeOptions) (*v1
 	volAnnotations["openEBSProvisionerIdentity"] = p.identity
 	volAnnotations["openebs.io/cas-type"] = casVolume.Spec.CasType
 
-	fsType, err := ParseClassParameters(options.Parameters)
```

Comment:
Change the name of this ParseClassParameter -> validateFSType that takes string as param. And validate the FS type returned in the VolumeSpec. 
Or remove the entire references to the ParseClassParameters .. and assume that a valid FS type is returned.

Reply:
Assuming valid FSType here.